Projekty/Projekt2/Gruszkowska_Gryszkiewicz_Chylak/KM1.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Gdy dokonujemy analizy, wyniki warto skomentować chociaż jednym zdaniem. Jupyter Notebook jest formą raportu - odbiorca powinien być w stanie zrozumieć analizy bez skrupulatnego wczytywania się w kod (odnosi się to także do kolejnych komórek)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>